.github/workflows/pull_request.yml

```diff
@@ -10,13 +10,16 @@ jobs:
     uses: swiftlang/github-workflows/.github/workflows/swift_package_test.yml@main
     with:
       linux_build_command: "swift build"
-      linux_exclude_swift_versions: '[{"swift_version": "5.8"}, {"swift_version": "5.9"}, {"swift_version": "5.10"}]'
+      linux_swift_versions: '["nightly-main"]'
       windows_build_command: "Invoke-Program swift build"
-      windows_exclude_swift_versions: '[{"swift_version": "5.9"}, {"swift_version": "5.10"}]'
+      windows_swift_versions: '["nightly-main"]'
       enable_macos_checks: true
       macos_xcode_versions: '["16.3"]'
```

Comment:
Not sure if you'd want to do this now or in a separate change, but it looks like Xcode 26.0 with a Swift 6.2 compiler is an option now: https://github.com/swiftlang/github-workflows/blob/2fe9a61880307905778b1718c95c28f92a90ad7e/.github/workflows/swift_package_test.yml#L12. Should we bump this version here?

Reply:
Oops. Merged too soon. I'll do this in a follow up for all three repos